src/main/java/com/commercetools/sync/producttypes/ProductTypeSync.java

```diff
@@ -89,10 +89,10 @@ public ProductTypeSync(@Nonnull final ProductTypeSyncOptions productTypeSyncOpti
      * Fetches existing {@link ProductType} objects from CTP project that correspond to passed {@code batch}.
      * Having existing product types fetched, {@code batch} is compared and synced with fetched objects by
      * {@link ProductTypeSync#syncBatch(List, List)} function. When fetching existing product types results in
-     * an empty optional then {@code batch} isn't processed.
+     * an empty {@link ProductTypeSyncStatistics} object then {@code batch} isn't processed.
```

Comment:
Something is wrong with this sentence =/

```
When fetching existing product types results in an empty {@link ProductTypeSyncStatistics} object then {@code batch} isn't processed.
```

What does `empty {@link ProductTypeSyncStatistics} object` mean?

Could you rephrase this sentence?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/316/commits/028db66424729292e46b4794433c16536b373927